src/AI-DataFrameInspector/AISpDataFramePresenter.class.st

```diff
@@ -0,0 +1,250 @@
+"
+Class for handling the dataframe in 'interactive' mode.
```

Comment:
It would be nice to add more info here. To explain some of the features it adds for example :)

Reply:
Done :)